plugins/gitea/plugin.go

```diff
@@ -0,0 +1,22 @@
+package gitea
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "gitea",
+		Platform: schema.PlatformInfo{
+			Name:     "Gitea",
+			Homepage: sdk.URL("https://gitea.com/"),
```

Comment:
We usually link to the project homepage here
```suggestion
			Homepage: sdk.URL("https://gitea.io/"),
```